zebra/rt_netlink.c

```diff
@@ -2021,6 +2021,21 @@ static int netlink_neigh_update(int cmd, int ifindex, void *addr, char *lla,
 	if (lla)
 		nl_attr_put(&req.n, sizeof(req), NDA_LLADDR, lla, llalen);
 
+	if (IS_ZEBRA_DEBUG_KERNEL) {
+		char ip_str[256];
+		struct interface *ifp =
+			if_lookup_by_index(ifindex, VRF_DEFAULT);
+		if (family == AF_INET6)
+			snprintfrr(ip_str, sizeof(ip_str), "ipv6 %pI6",
+				   (struct in6_addr *)addr);
+		else
+			snprintfrr(ip_str, sizeof(ip_str), "ipv4 %pI4",
+				   (in_addr_t *)addr);
+		zlog_debug("%s: %s ifname %s ifindex %u addr %s mac %pEA",
```

Comment:
Might be useful to print VRF too?

Reply:
Thanks for your comments.
Addressed review comments in 0f9a3d765215baf8f0513a0ae2ad8b6157c31b7c

Output:
netlink_neigh_update: RTM_DELNEIGH ifname vet
h0 ifindex 5 addr ipv4 169.254.0.1 mac 82:96:20:33:25:1f vrf default(0)